packages/core/src/api/toUnits.ts

```diff
@@ -0,0 +1,43 @@
+import type { Calculator, Dinero, Transformer } from '../types';
+import { isArray, getDivisors } from '../utils';
+
+export type ToUnitsParams<TAmount, TOutput> = readonly [
+  dineroObject: Dinero<TAmount>,
+  transformer?: Transformer<TAmount, TOutput, readonly TAmount[]>
+];
+
+export function toUnits<TAmount, TOutput>(calculator: Calculator<TAmount>) {
+  const getDivisorsFn = getDivisors(calculator);
+
+  return function toUnitsFn(
+    ...[
+      dineroObject,
+      transformer = ({ value }) => value as TOutput,
```

Comment:
I don't like the `as` in there. I've been thinking about these functions a lot and the return of the default transformer isn't supposed to be `TOutput` with how the declaration written.  

If a `transformer` isn't provided then return the correct value that is intended and not use a default argument and infer `TOutput`.

So just return the `TAmount[]` in `toUnits` and in `string` in `toDecimal`. It doesn't have to go through a default `transformer`. Leaving the `as` in there just opens up the possibility of new issues in the future during refactors.

```suggestion
      transformer,
```